index.html

```diff
@@ -60,11 +60,11 @@
                         <li class="nav-item dropdown active" style="margin-left: 50px;">
                             <a class="nav-link dropdown-toggle" href="#" id="dropdownId" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Available furniture</a>
                             <div class="dropdown-menu" aria-labelledby="dropdownId">
-                                <a class="dropdown-item" href="#">Office chairs</a>
-                                <a class="dropdown-item" href="#">Dinner chairs</a>
-                                <a class="dropdown-item" href="#">Rocking chairs</a>
-                                <a class="dropdown-item" href="#">Glider chairs</a>
-                                <a class="dropdown-item" href="#">Sofa</a>
+                                <a class="dropdown-item" href="Category/Chair type/Officechair-listpage.html">Office chairs</a>
+                                <a class="dropdown-item" href="Category/Chair type/Officechair-listpage.html">Dinner chairs</a>
```

Comment:
Same file??

Reply:
yes,  all dropdow items will lead to same office chair category page. I will add the link for the other.